pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -303,9 +315,15 @@ func createDVSecret(creds provider.DataVolumeCredentials, vmImport *v2vv1alpha1.
 	}
 }
 
-func shouldValidate(vmi *v2vv1alpha1.VirtualMachineImportStatus) bool {
-	// TODO: check the status - status manipulation package is needed
-	return true
+func shouldValidate(vmiStatus *v2vv1alpha1.VirtualMachineImportStatus) bool {
+	validingCondition := conditions.FindConditionOfType(v2vv1alpha1.Validating, vmiStatus.Conditions)
```

Comment:
Should `validingCondition`  be `validatingCondition` or it is just an abbreviation of it ?